setup_for_python_metadata.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup
+import sys
+
+# This is a simple and fragile way of passing the current version
+# from cmake to setup as I assume no one else will use this.
+#
+# This scprit has to have the version always specified as last argument.
+version = sys.argv.pop()
+
+setup(
+    name='createrepo_c',
+    description='C implementation of createrepo',
+    version=version,
+    license='GPLv2+',
+    author='RPM Software Management',
+    author_email='rpm-ecosystem@lists.rpm.org',
+    url='https://github.com/rpm-software-management',
+    classifiers=(
+        'License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
+        'Operating System :: POSIX :: Linux',
+        'Programming Language :: C',
+        'Topic :: System :: Software Distribution',
+        'Topic :: System :: Systems Administration',
+        "Programming Language :: Python :: 2",
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+    ),
+    packages=['createrepo_c'],
+    package_dir={
+        'createrepo_c': 'src/python/createrepo_c'
+    },
+    entry_points={
+        'console_scripts': [
+            'createrepo_c=createrepo_c:createrepo_c',
+            'mergerepo_c=createrepo_c:mergerepo_c',
+            'modifyrepo_c=createrepo_c:modifyrepo_c',
+            'sqliterepo_c=createrepo_c:sqliterepo_c'
+        ]
```

Comment:
Why are we generating entry points? Won't these overwrite the ones generated by CMake?

Reply:
I just blindly copied them from the original setup.py. Thank you for catching that.